anker/src/instruction.rs

```diff
@@ -287,7 +287,7 @@ accounts_struct! {
             is_signer: false,
             is_writable: false,
         },
-        pub rewards_destination {
+        pub ust_reserve {
```

Comment:
<del>Wait, but now we reference two UST accounts: `ust_token` and `ust_reserve`. Shouldn’t that be one account?</del><ins>Ah, of course, the two `_token` accounts are the ones owned by the pool, and two are the ones owned by us.</ins>

Maybe we can call the ones owned by the pool `st_sol_pool_account` and `ust_pool_account`, to distinguish them from our `st_sol_reserve_account` and `ust_reserve`? (Should we also call it `_account`? And let’s put them next to each other, it doesn’t really matter for us with named accounts, but it matters for what you see on block explorers.)